Lib/test/test_buffer.py

```diff
@@ -4412,6 +4412,24 @@ def test_issue_7385(self):
         x = ndarray([1,2,3], shape=[3], flags=ND_GETBUF_FAIL)
         self.assertRaises(BufferError, memoryview, x)
 
+    # Test PyBuffer_SizeFromFormat()
+    @support.cpython_only
+    def test_pybuffer_size_from_format(self):
+        from ctypes import pythonapi
+        PyBuffer_SizeFromFormat = pythonapi.PyBuffer_SizeFromFormat
```

Comment:
This test should be able to run even if `ctypes` is not compiled. Could you expose this function in `Modules/_testcapimodule.c`?